packages/skeleton-common/src/classes/steps.ts

```diff
@@ -0,0 +1,19 @@
+import { defineSkeletonClasses } from '../internal/define-skeleton-classes.js' with { type: 'macro' };
+
+export const classesSteps = defineSkeletonClasses({
+	root: '',
+	list: 'flex items-center gap-2',
+	item: 'flex items-center gap-2 grow',
+	trigger: 'flex items-center gap-1',
+	indicator: [
+		'size-8 rounded-full preset-tonal content-center',
+		'data-complete:preset-filled-primary-500',
+		'data-current:preset-filled-primary-500',
+	],
+	itemText: '',
+	separator: 'h-0 border-t border-t-surface-200-800 grow',
+	content: '',
+	control: '',
+	prevTrigger: 'btn preset-filled',
```

Comment:
One of the things we've learned with the new `class` attribute approach in v4 is that it's hard to remove/replace complex styles like `btn` and `preset` classes. If the provided class doesn't overwrite all base styles, then you'll get a weird mix of the two shown.

We may in fact opt to some of these for components in future versions Skeleton. For now, let's remove these styles from this file, and instead implement them in user-land. Like so:

```html
<Steps.PrevTrigger class="btn preset-filled">Previous</Steps.PrevTrigger>
```

The idea being that someone might want to use an icon and different preset combos:

```html
<Steps.PrevTrigger class="btn-icon preset-tonal hover:preset-filled">&larr;</Steps.PrevTrigger>
```

Reply:
Removed default styles from prevTrigger and nextTrigger